### ISSUE TITLE
Support `<>` bracket

### ISSUE THREAD

**Issue by brizaldi** [tearf001/vscode-flutter-code-select#12]
Hi, just wondering, it would be really helpful if this extension also support this kinda bracket `<>`

**Comment by Mativve** [comment]
Yeah! That would be cool. Im hope this functionality is added too in the future.
